woocommerce/google-listings-and-ads
# Includes removed campaigns in Program Reports

### Changes proposed in this Pull Request:



As part of the migration process of SSC campaigns to PMax campaigns, users need to be able to compare removed SSC campaigns with new campaigns (PMax). This PR includes removed campaigns in the programs reports filter. 

See more context in the following links:

- [Design Proposal i2: Performance Max Migration (Milestone 2)](https://woodesigngrowconsultants.wordpress.com/2022/04/20/design-proposal-i2-performance-max-migration-milestone-2/#in-reports)
- [Figma File](https://href.li/?https://www.figma.com/file/eQ9O4m2flzrcAiDMXkOW0m/Google-Listings-%26-Ads-v1.x?node-id=7840%3A127644)

Also, I added two small corrections to CSS linting.

### Screenshots:
![image](https://user-images.githubusercontent.com/2488994/168565091-e32db514-cbc4-4b66-850b-477541015e9d.png)


### Detailed test instructions:


1.  Go to GLA->Reports->Programs
2.  Show -> Single Program

![image](https://user-images.githubusercontent.com/2488994/168544470-8a18481b-32d3-4e9a-b33f-75cf478bd75b.png)

3. Search for a removed campaign or go to your dashboard to delete one campaign and check if the filter shows it.

![image](https://user-images.githubusercontent.com/2488994/168544616-70f0e425-c62f-4b76-873b-a101dfe3481f.png)

### Additional details:

The suffix "(old)" is added if:

- the campaign is an SSC campaign.
- it has been removed.
- The user has been migrated (all SSC campaigns are removed and only PMax campaigns are available). This can be set manually if you add a new option: `gla_campaign_convert_status ` with the following value: `a:2:{s:6:"status";s:9:"converted";s:7:"updated";i:1652687806;}`

### Changelog entry

>Add - includes removed campaign in the program report section
